src/main/webapp/app/assessment/assessment-general-feedback/assessment-general-feedback.component.ts

```diff
@@ -22,4 +22,16 @@ export class AssessmentGeneralFeedbackComponent {
         this.feedbackClone.detailText = trimmedText.length > 0 ? trimmedText : null;
         this.feedbackChange.emit(this.feedbackClone);
     }
+    allowDrop(ev: any) {
```

Comment:
can you write event. Do we have a reasonable type?

Reply:
Yes, DragEvent so I changed it accordingly, thanks for pointing this out.